tests/test.bats

```diff
@@ -258,3 +263,31 @@ _healthcheck_wait ()
 	fin rm -f
 	rm -f .docksal/docksal-local.env
 }
+
+@test "Check Platform.sh Integration" {
+	[[ $SKIP == 1 ]] && skip
+
+	### Setup ###
+	docker rm -vf "$NAME" >/dev/null 2>&1 || true
+	docker run --name "$NAME" -d \
+		-v /home/docker \
+		-v $(pwd)/../tests:/var/www \
+		-e SECRET_PLATFORMSH_CLI_TOKEN \
+		"$IMAGE"
+	_healthcheck_wait
+
+	### Tests ###
+	run docker exec -it -u docker "$NAME" bash -c 'source $HOME/.docksalrc >/dev/null 2>&1; echo "${SECRET_PLATFORMSH_CLI_TOKEN}"'
```

Comment:
If you want to test the both this and the one below `run` returns something, then they should be tested independently. Also, should test for a specific value in the output to be 100% sure.